sapl/webpack-stats.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"status":"done","publicPath":"/static/sapl/","chunks":{"chunk-vendors":[{"name":"css/chunk-vendors.81e1a0b9.css","publicPath":"/static/sapl/css/chunk-vendors.81e1a0b9.css","path":"../sapl/sapl/static/sapl/css/chunk-vendors.81e1a0b9.css"},{"name":"js/chunk-vendors.479a02b6.js","publicPath":"/static/sapl/js/chunk-vendors.479a02b6.js","path":"../sapl/sapl/static/sapl/js/chunk-vendors.479a02b6.js"},{"name":"css/chunk-vendors.81e1a0b9.css.map","publicPath":"/static/sapl/css/chunk-vendors.81e1a0b9.css.map","path":"../sapl/sapl/static/sapl/css/chunk-vendors.81e1a0b9.css.map"}],"compilacao":[{"name":"css/compilacao.3372b760.css","publicPath":"/static/sapl/css/compilacao.3372b760.css","path":"../sapl/sapl/static/sapl/css/compilacao.3372b760.css"},{"name":"js/compilacao.e49c92ad.js","publicPath":"/static/sapl/js/compilacao.e49c92ad.js","path":"../sapl/sapl/static/sapl/js/compilacao.e49c92ad.js"},{"name":"css/compilacao.3372b760.css.map","publicPath":"/static/sapl/css/compilacao.3372b760.css.map","path":"../sapl/sapl/static/sapl/css/compilacao.3372b760.css.map"}],"global":[{"name":"css/global.a9fa7d26.css","publicPath":"/static/sapl/css/global.a9fa7d26.css","path":"../sapl/sapl/static/sapl/css/global.a9fa7d26.css"},{"name":"js/global.720ff058.js","publicPath":"/static/sapl/js/global.720ff058.js","path":"../sapl/sapl/static/sapl/js/global.720ff058.js"},{"name":"css/global.a9fa7d26.css.map","publicPath":"/static/sapl/css/global.a9fa7d26.css.map","path":"../sapl/sapl/static/sapl/css/global.a9fa7d26.css.map"}],"painel":[{"name":"css/painel.baa845ab.css","publicPath":"/static/sapl/css/painel.baa845ab.css","path":"../sapl/sapl/static/sapl/css/painel.baa845ab.css"},{"name":"js/painel.a7f54dae.js","publicPath":"/static/sapl/js/painel.a7f54dae.js","path":"../sapl/sapl/static/sapl/js/painel.a7f54dae.js"},{"name":"css/painel.baa845ab.css.map","publicPath":"/static/sapl/css/painel.baa845ab.css.map","path":"../sapl/sapl/static/sapl/css/painel.baa845ab.css.map"}],"sessao_online":[{"name":"css/sessao_online.ddd218af.css","publicPath":"/static/sapl/css/sessao_online.ddd218af.css","path":"../sapl/sapl/static/sapl/css/sessao_online.ddd218af.css"},{"name":"js/sessao_online.5a4e18dd.js","publicPath":"/static/sapl/js/sessao_online.5a4e18dd.js","path":"../sapl/sapl/static/sapl/js/sessao_online.5a4e18dd.js"},{"name":"css/sessao_online.ddd218af.css.map","publicPath":"/static/sapl/css/sessao_online.ddd218af.css.map","path":"../sapl/sapl/static/sapl/css/sessao_online.ddd218af.css.map"}]}}
```

Comment:
qual a necessidade desse arquivo?

Reply:
é a ponte entre entre sapl e sapl-frontend... esse arquivo já está na 144... não tem nada a haver com django-channels... é por ele que o django, através do webpack_loader, sabe o que do sapl-frontend tem pra ser carregado

Reply:
complementando... durante a implementação de #2486 fiz mudanças no sapl-frontend para comunicar com websocks... toda vez que se mexe no frontend é gerada uma nova versão e add ao sapl... seja qual for obranch do sapl... essas mudanças na pasta static e neste arquivo webpack_stats.json, são irrelevantes ao desenvolvimento rotineiro do backend... no mais... quando estabilizarmos frontend, django-chanels e solr, pretendo colocar em discussão o retorno do frontend para dentro do projeto sapl e as inclusões do node e yarn dentro do docker para gerar o build do frontend internamente e o histórico do sapl seguir como um só projeto.